includes/class-wc-gateway-amazon-payments-advanced-abstract.php

```diff
@@ -556,15 +556,15 @@ public function get_form_fields() {
 			'mini_cart_button'              => array(
 				'title'       => __( 'Amazon Pay on mini cart', 'woocommerce-gateway-amazon-payments-advanced' ),
 				'label'       => __( 'Enable Amazon Pay on mini cart', 'woocommerce-gateway-amazon-payments-advanced' ),
-				'description' => __( 'This will only work if you are using WooCommerce\'s mini cart. If you enable it and the Amazon Pay does not show please disable since it also enables loading of required assets globally in your frontend. Not compatible with WooCommerce Blocks.', 'woocommerce-gateway-amazon-payments-advanced' ),
+				'description' => __( 'This will only work if you are using WooCommerce\'s mini cart. If you enable it and the Amazon Pay does not show please disable since it also enables loading of required assets globally in your frontend. Compatible WooCommerce Blocks.', 'woocommerce-gateway-amazon-payments-advanced' ),
```

Comment:
Minor and pitty comment 😛
'Compatible WooCommerce Blocks.' --> 'Compatible with WooCommerce Blocks.'